engine/include/cubos/engine/ui/canvas/vertical_stretch.hpp

```diff
@@ -0,0 +1,25 @@
+/// @file
+/// @brief Component @ref cubos::engine::VerticalStretch.
+/// @ingroup canvas-plugin
+
+#pragma once
+
+#include <glm/glm.hpp>
+
+#include <cubos/core/gl/render_device.hpp>
+#include <cubos/core/reflection/reflect.hpp>
+
+#include <cubos/engine/api.hpp>
+
+namespace cubos::engine
+{
+    /// @brief Component which makes a UI element fit vertically to its parent.
+    /// @ingroup canvas-plugin
+    struct CUBOS_ENGINE_API VerticalStretch
```

Comment:
Might be a good idea to name this and the other component `UI*Stretch`, seems a bit too generic of a name (will also make it easier to find in component drop downs).